site/public/js/server.js

```diff
@@ -182,8 +182,12 @@ function newDeleteCourseMaterialForm(id, file_name, str_id = null) {
 
     $('.popup-form').css('display', 'none');
     var form = $("#delete-course-material-form");
-    $('.delete-course-material-message', form).html('');
-    let cm_message = '<b>'+file_name+'</b>';
+    var deleteMessageElement = form.find('.delete-course-material-message')[0];
```

Comment:
It's generally better to use `let` and `const` instead of `var` in JS due to `var`'s loose scoping rules.

```suggestion
    const deleteMessageElement = form.find('.delete-course-material-message')[0];
```